idx/memory/memory.go

```diff
@@ -639,42 +597,75 @@ func (m *MemoryIdx) Prune(orgId int, oldest time.Time) ([]idx.Archive, error) {
 	return pruned, nil
 }
 
-// filepath.Match doesn't support {} because that's not posix, it's a bashism
-// the easiest way of implementing this extra feature is just expanding single queries
-// that contain these queries into multiple queries, who will be checked separately
-// and whose results will be ORed.
```

Comment:
As a first reader of this code, this comment confused me. I read it as "We use filepath.Match, but need to workaround it's lack of support for `{}`". I could change it to:
"We don't use filepath.Match because it..."